compiler/rustc_infer/src/infer/combine.rs

```diff
@@ -809,6 +831,10 @@ impl<'tcx> TypeRelation<'tcx> for ConstInferUnifier<'_, 'tcx> {
         true
     }
 
+    fn mark_ambiguous(&mut self) {
+        bug!()
```

Comment:
that should be reachable when using generic const exprs when relating a const inference var (from a const param) with an unevaluated constant during coherence